src/main/scala/io/qbeast/spark/delta/DeltaQbeastSnapshot.scala

```diff
@@ -180,8 +183,10 @@ case class DeltaQbeastSnapshot(protected override val snapshot: Snapshot)
 
   /**
    * Loads the dataset of qbeast blocks for a given revision
-   * @param revisionID the revision identifier
-   * @return the Dataset of QbeastBlocks
+   * @param revisionID
+   *   the revision identifier
+   * @return
+   *   the Dataset of QbeastBlocks
    */
   def loadRevisionBlocks(revisionID: RevisionID): Dataset[AddFile] = {
```

Comment:
We should rename `loadRevisionBlocks` as well as `loadStagingBlocks` since `Block` and `AddFile` no longer have a one-to-one mapping.

I suggest something along the lines of `longRevisionAddFiles`.

Reply:
Sounds good. Should the name be reduced to "files" instead of using the AddFiles? 

I would rename the `loadStagingBlocks` for `loadStagingFiles` as well.

Reply:
Agree